lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -292,7 +292,7 @@ def user_alter(cursor, module, user, password, role_attr_flags, encrypted, expir
         # Do we actually need to do anything?
         pwchanging = False
         if password is not None:
-            if encrypted:
+            if encrypted == 'ENCRYPTED':
                 if password.startswith('md5'):
```

Comment:
`encrypted` name is used for a variable and a module parameter but the variable and the module parameter don't have the same type, that's error-prone.

Another possible fix would be to use a boolean for [encrypted](https://github.com/acasademont/ansible/blob/751e8edcc0a883a218617dc3b85395ab98c8bafe/lib/ansible/modules/database/postgresql/postgresql_user.py#L679) variable (this would require to update the way the `WITH` clause is built in `add_user` and `alter_user` methods).